ios/DevKit/DevKit/Classes/Validation/FormValidator.swift

```diff
@@ -35,6 +35,7 @@ public class FormValidator {
     ///   - validators: The validators for your password.
     public func isValid(_ password: String, validators: [ValidationType]) -> Bool {
         return isValid(password, passwordRegex: ValidationType.getRegexCode(with: validators))
+               || ValidationType.checkForOnlyMinCharacters(password, validators: validators)
```

Comment:
We should update our tests to cover for this case

Reply:
will do 👍